### ISSUE TITLE
Need Help After Update

### ISSUE THREAD

**Issue by nuc3boys** [nextcloud/vm#1177]
## Steps to reproduce
1.Made several Attempts to update with same result
2.
3.

## Expected behaviour
Normally it updates and all is well

## Actual behaviour
Nextcloud isn't functioning

## Server configuration

**Nextcloud server version:** (see your admin page)
18.0.3
**Server OS** (Ubuntu server is default)
Win 10 Hyper-V
**How did you install the VM?** (Scripted install from master **OR** Released version)
Your VM
## Network
**Do you use DHCP?**

**Is port 80 and/or 443 open?**
Yes
#### Log file (/var/log/nextcloud/nextcloud.log)
```
Insert your nextcloud.log file here
```

#### Installation log (command output)
```
Insert the CLI output here
```

#### Screenshoots

![image](https://user-images.githubusercontent.com/52842650/79390093-f63d0b00-7f23-11ea-852f-7658e690a70b.png)

**Comment by enoch85** [comment]
Hey,

Thanks for your post!

If you need hands on support, please [check this out](https://shop.hanssonit.se/product/premium-support-per-30-minutes/)

**Comment by nuc3boys** [comment]
Please forgive my ignorance here...but it appears the automated update script provided by you broke my cloud. Now I should pay to have it fixed?

**Comment by nuc3boys** [comment]
It appears the error in the script has been fixed. Thank you!

**Comment by enoch85** [comment]
> It appears the error in the script has been fixed. 

There was no commit done to the update script for a few weeks, this was probably a pure Nextcloud issue. I noticed the same thing today actually when updating a server. Then it suddenly just started to work.

Regarding you last comment; I don't know if you got the the paid version or not, but in any case I've spent 5 years on this so that people can enjoy Nextcloud without any Linux skills. Sometimes servers acts up, and if you want an expert to have a look rather than fixing it yourself there's a support option that I provide, as a service.

I'm glad it's all sorted anyway! :+1:

**Comment by nuc3boys** [comment]
Yes sir, I do have the paid version I purchased from your site....twice.

I admit I have very limited Linux skills which is precisely why I reached out to you and purchased your Hyper-V product. Which, by the way, is outstanding. It indeed is very simple for us novices to get a working NextCloud instance up and running with your “package”.

However, people like me who have limited Linux knowledge rely heavily on developers such as yourself for a bit of advice when things go wrong, (or we freak out because something changed). I don’t know what part of all that happens in the background during updates you have a hand in nor do I have a clue how it all works. But, I don’t want to know. I have no interest in learning which is why I gladly bought your product. My skills lay elsewhere.

I am fully aware you’re running a business, (one which I am a repeat customer of), and I am fully aware of your involvement with Nextcloud and the Nextcloud community at large.

Your latest response should have been your first response. Not, “buy my support” right out of the gate but a bit of free advice such as you just posted. 

Take no offense to this because it is not intended to be offensive, try not to come off so pretentious. I know you know your stuff and have worked hard, so do most in the Nextcloud community; so try and relax a little. 

As always, thanks for the help. It is appreciated. 

Sent Via Mobile
